mne/preprocessing/nirs/_tddr.py

```diff
@@ -47,29 +52,34 @@ def temporal_derivative_distribution_repair(raw):
     return raw
 
 
+# provide a short alias
+tddr = temporal_derivative_distribution_repair
+
+
 # Taken from https://github.com/frankfishburn/TDDR/ (MIT license).
 # With permission https://github.com/frankfishburn/TDDR/issues/1.
-# The only modification is the name,  scipy signal import and flake fixes.
+# The only modification is the name, scipy signal import and flake fixes.
 def _TDDR(signal, sample_rate):
-    # This function is the reference implementation for the TDDR algorithm for
-    #   motion correction of fNIRS data, as described in:
-    #
-    #   Fishburn F.A., Ludlum R.S., Vaidya C.J., & Medvedev A.V. (2019).
-    #   Temporal Derivative Distribution Repair (TDDR): A motion correction
-    #   method for fNIRS. NeuroImage, 184, 171-179.
-    #   https://doi.org/10.1016/j.neuroimage.2018.09.025
-    #
-    # Usage:
-    #   signals_corrected = TDDR( signals , sample_rate );
-    #
-    # Inputs:
-    #   signals: A [sample x channel] matrix of uncorrected
-    #            optical density data
-    #   sample_rate: A scalar reflecting the rate of acquisition in Hz
-    #
-    # Outputs:
-    #   signals_corrected: A [sample x channel] matrix of corrected
-    #   optical density data
+    """Correct data using the TDDR algorithm for motion correction.
+
+    The algorithm is described in:
+    Fishburn F.A., Ludlum R.S., Vaidya C.J., & Medvedev A.V. (2019).
+    Temporal Derivative Distribution Repair (TDDR): A motion correction
+    method for fNIRS. NeuroImage, 184, 171-179.
+    https://doi.org/10.1016/j.neuroimage.2018.09.025
+
+    Parameters
+    ----------
+    signal : ndarray
+        Uncorrected optical density data (sample x channel).
+    sample_rate : float
+        Sampling rate in Hz.
+
+    Returns
+    -------
+    signal_corrected : ndarray
+        Corrected optical density data (sample x channel).
+    """
```

Comment:
Please revert this, the old code was closer to copy-paste from the upstream repo which we'd like to preserve (easier for updating, etc.), see comment above about only modifications...

Reply:
I didn't know that there was an upstream .py file - I thought @rob-luke ported MATLAB code to Python. OK, I'll revert.